sdk/communication/Azure.Communication.PhoneNumbers/src/PhoneNumbersClient.cs

```diff
@@ -92,7 +93,7 @@ private PhoneNumbersClient(string endpoint, HttpPipeline httpPipeline, PhoneNumb
         /// <param name="apiVersion"> Api Version. </param>
         private PhoneNumbersClient(ClientDiagnostics clientDiagnostics, HttpPipeline pipeline, string endpoint, string acceptedLanguage, string apiVersion = "2021-03-07")
         {
-            InternalClient = new InternalPhoneNumbersClient(clientDiagnostics, pipeline, endpoint, apiVersion);
+            InternalClient = new InternalPhoneNumbersClient(clientDiagnostics, pipeline, new Uri(endpoint), apiVersion);
```

Comment:
Why is endpoint replaced with new Uri(endpoint)? Did the constructor change?

Reply:
yes, I'm not sure why, but after doing the generation this was failing, seems like a simple fix